src/cordova/build_plugin.action.mjs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from 'url';
+
+import {cpSync, mkdirSync} from 'fs';
+import path from 'path';
+import rmfr from 'rmfr';
+import {execSync} from 'child_process';
+
+/**
+ * @description Builds the parameterized cordova binary (ios, macos, android).
+ *
+ * @param {string[]} parameters
+ */
+export async function main(...parameters) {
+  const PLATFORM = parameters[0];
+
+  const PLUGIN_OUTPUT = path.join(process.env.BUILD_DIR, 'cordova', 'plugin');
+  await rmfr(PLUGIN_OUTPUT);
+  mkdirSync(PLUGIN_OUTPUT, {recursive: true});
```

Comment:
*nit* we can also use the async version by `import { mkdir } from 'node:fs/promises';`

Reply:
Done